code/game/objects/items/ego_weapons/non_abnormality/cane.dm

```diff
@@ -12,7 +12,6 @@
 	var/charge_effect = "deal an extra attack in damage."
 	var/charge_cost = 2
 	var/charge
-	var/activated
```

Comment:
Leave this one in, These weapons were originally going to have a buff applied (and thus I would need to know if it was applied), and if I ever get around to it, they will in the future